pkg/annotations.go

```diff
@@ -36,23 +36,31 @@ const (
 
 	// AnnotationEnvironment is the environment name that should be sent to Cronitor
 	// for the CronJob.
-	// Optional. Overrides the default chart-wide environment if present.
```

Comment:
I could add this back...I removed it since all the annotations are optional, and didn't want to imply some were required.

Reply:
That's a fair point